test/unit/dropbox.js

```diff
@@ -51,4 +93,23 @@ describe('Dropbox', () => {
       chai.assert.equal(dbx.pathRoot, undefined);
     });
   });
+
+  describe('setCommonHeaders', () => {
```

Comment:
Should this also test combinations of these parameters as well?

Reply:
Yea that's a good point